Content.Server/StationEvents/PowerGridCheck.cs

```diff
@@ -45,9 +45,10 @@ public override void Startup()
             _failDuration = IoCManager.Resolve<IRobustRandom>().Next(30, 120);
             var componentManager = IoCManager.Resolve<IComponentManager>();
             
-            foreach (var component in componentManager.EntityQuery<PowerReceiverComponent>())
+            foreach (PowerReceiverComponent component in componentManager.EntityQuery<PowerReceiverComponent>())
             {
                 component.PowerDisabled = true;
+                _powered.Add(component.Owner);
```

Comment:
Couldn't you just do _powered[component.Owner] = component.PowerDisabled; so disabled stuff at least doesn't get turned back on for now.

Reply:
Ah, I think that's what you meant to do originally with the unused list. Does anything else actually set the disabled flag right now, though?